<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix tab_name="plot" rows="3" columns="1">
   <plot row="0" col="0">
    <range top="0.827624" bottom="-0.999499" left="0.000000" right="98.998246"/>
    <limitY/>
    <curve custom_transform="noTransform" name="/velocity/body_level_frame/twist/twist/linear/x" R="0" G="0" B="0"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/sigma_minus.0" R="234" G="0" B="185"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/sigma_plus.0" R="255" G="0" B="183"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/x_dot" R="255" G="0" B="3"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range top="1.128378" bottom="-1.142109" left="0.000000" right="98.998246"/>
    <limitY/>
    <curve custom_transform="noTransform" name="/velocity/body_level_frame/twist/twist/linear/y" R="0" G="0" B="0"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/sigma_minus.1" R="255" G="0" B="180"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/sigma_plus.1" R="255" G="0" B="255"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/xy_plus/y_dot" R="255" G="0" B="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="2" col="0">
    <range top="3.271476" bottom="-3.962226" left="0.000000" right="98.998246"/>
    <limitY/>
    <curve custom_transform="noTransform" name="/velocity/body_level_frame/twist/twist/linear/z" R="0" G="0" B="0"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/z_plus/sigma_minus.1" R="254" G="0" B="167"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/z_plus/sigma_plus.1" R="255" G="0" B="179"/>
    <curve custom_transform="noTransform" name="/xyz_debug_estimate/z_plus/z_dot" R="255" G="0" B="0"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <DataLoad_ULog>
   <no_params/>
  </DataLoad_ULog>
  <ROS_Topic_Streamer>
   <selected_topics list="/velocity/body_level_frame;/xyz_debug_estimate;/rgbd_to_velocity/body_level_frame"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
</root>

